app/src/main/java/org/ergoplatform/android/ErgoFacade.kt

```diff
@@ -31,6 +32,16 @@ fun formatErgsToString(ergs: Float, context: Context): String {
     return DecimalFormat(context.getString(R.string.format_erg)).format(ergs).replace(',', '.')
 }
 
+fun formatFiatToString(amount: Float, currency: String, context: Context): String {
+    return DecimalFormat(context.getString(R.string.format_fiat)).format(amount).replace(',', '.') +
+            " " + currency.toUpperCase(Locale.getDefault())
+}
+
+fun formatLongToFloatWithDecimals(amount: Long, decimals: Int): String {
+    val valueToShow: Float = (amount.toFloat()) / (10f.pow(decimals))
```

Comment:
`Long.MAX_VALUE` can be converted to `Float` - accuracy is lower though, but I think it is okay here because we use this only for displaying purposes. Big numbers aren't displayed very user-friendly though, so I am going to change this to a 1.0K/1.0M/1.0B representation for big numbers.